File: 프로그래머스/unrated/155652. 둘만의 암호/둘만의 암호.java
class Solution {
    public String solution(String s, String skip, int index) {
        char[] chars = s.toCharArray();
        for (int i = 0; i < chars.length; i++) {
            char c = chars[i];
            int count = 0;
            while (count < index) {
                c = (char) (c == 'z' ? 'a' : c + 1);
                if (skip.indexOf(c) == -1) {
                    count++;
                }
            }
            chars[i] = c;
        }
        return new String(chars);
    }
}

File: 프로그래머스/unrated/155652. 둘만의 암호/둘만의 암호.kt
class Solution {
    fun solution(s: String, skip: String, index: Int): String {
        val chars = s.toCharArray()
    for (i in chars.indices) {
        var c = chars[i]
        var count = 0
        while (count < index) {
            c = if (c == 'z') 'a' else c + 1
            if (skip.indexOf(c) == -1) {
                count++
            }
        }
        chars[i] = c
    }
    return String(chars)
    }
}